styles/style.css

```diff
@@ -0,0 +1,340 @@
+
+/* General settings for web browsers */
+@font-face {
+  font-family: "Inter";
+  src: url(../fonts/Inter.ttf);
+}
+
+*,
+*::after,
+*::before {
+  border: 0;
+  margin: 0;
+  padding: 0;
+  font-family: 'Franklin Gothic Medium', 'Arial Narrow', Arial, sans-serif;
+}
+
+/* Mobile Version of screen smaller than considered ( < 375px) */
+@media (max-width: 375px) {
+  body {
+    /* Nothing to display for the moment */
+  }
+}
+
+/* Mobile version for phone with considered screen size minimum 375px */
+@media (min-width: 375px) {
+  .toolbar {
+    position: fixed;
+    left: 1vw;
+    top: 1vw;
+    z-index: 10;
+    width: 98vw;
+    height: 62px;
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    background: white;
+  }
+
+  .toolbar .logo {
+    height: 48px;
+    width: 24px;
+    margin-left: 25px;
+
+    /* content */
+    display: flex;
+    align-items: center;
+    font-family: 'Inter', 'Franklin Gothic Medium', 'Arial Narrow', Arial, sans-serif;
+    font-style: normal;
+    font-weight: bolder;
+    font-size: 25px;
+    letter-spacing: 3px;
+    color: #36b37e;
+  }
+
+  .toolbar .menu {
+    height: 40px;
+    width: 40px;
+    padding: 8px;
+    margin-right: 25px;
+    margin-right: 10px;
+    border-radius: 8px;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+  }
+
+  .menu img {
+    width: 18px;
+    height: 12px;
+  }
+
+  /* Headline section */
+  .headline {
+    width: 100vw;
+    height: 88vh;
+    position: relative;
+    top: 60px;
+    background-image: url(../images/header-illsutration-mobile@2x.png);
+    background-size: contain;
+    background-repeat: no-repeat;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+  }
```

Comment:
- Good job adding the image as a background image using CSS, but the background image for this section is not properly set. You can fix this by using `cover` instead of `contain` for the `background-size` property. Please fix this 🙏 

Your project:
![wgwegwgw](https://user-images.githubusercontent.com/73607512/235646602-2f6da8e2-8b95-453e-b73e-349eac4a3e6b.PNG)